.github/workflows/diffs.yml

```diff
@@ -0,0 +1,346 @@
+---
+name: create-diffs
+on:
+    pull_request:
+        types: ["opened", "synchronize"]
+jobs:
+    tpgb:
+        runs-on: ubuntu-latest
+        name: "Build TPGB"
+        env:
+            GOPATH: /home/runner/work/magic-modules/go
+            GO111MODULE: auto
+        steps:
+            - uses: actions/checkout@v1
+            - uses: actions/setup-go@v1
+              with:
+                  go-version: 1.11.x
+            - uses: actions/setup-ruby@v1
+              with:
+                  ruby-version: 2.x
+                  architecture: 'x64'
+            - run: go get golang.org/x/tools/cmd/goimports
+            - run: bundle install
+            - uses: actions/checkout@v1
+              with:
+                  path: go/src/github.com/terraform-providers/terraform-provider-google-beta
+                  repository: terraform-providers/terraform-provider-google-beta
+                  ref: master
+
+            - name: Generate new commit.
+              run: |
+                  git config --global user.email "magic-modules@google.com" && git config --global user.name "Modular Magician"
+                  pushd /home/runner/work/magic-modules/go/src/github.com/terraform-providers/terraform-provider-google-beta
+                  find . -type f -not -wholename "./.git*" -not -wholename "./vendor*" -not -name ".travis.yml" -not -name ".golangci.yml" -not -name "CHANGELOG.md" -not -name "GNUmakefile" -not -name "docscheck.sh" -not -name "LICENSE" -not -name "README.md" -not -wholename "./examples*" -not -name "go.mod" -not -name "go.sum" -not -name "staticcheck.conf" -not -name ".go-version" -not -name ".hashibot.hcl" -not -name "tools.go"  -exec git rm {} \;
+                  popd
+                  export PATH=$PATH:$GOPATH/bin
```

Comment:
Can we do this earlier on in our steps, or in the image we use?

Reply:
Same as above - could do, but this is a little better for us in the long run.

Reply:
my nitpicky followup is to ask we do it at the start of the `run` in that case, right before/after the `git config` step & before we modify stuff